src/library_syscall.js

```diff
@@ -19,7 +19,7 @@ var SyscallsLibrary = {
 
     // global state
     mappings: {},
-    umask: 0x1FF,  // S_IRWXU | S_IRWXG | S_IRWXO
```

Comment:
I think JS supports octal values so could instead just write O22 here which might be more readable.

I'm surprised this didn't effect any tests.  Might be worth adding one?

Reply:
Okay, I changed it to octal representation using the modern syntax for octals. I'm not sure how to add a test for this though.

Reply:
Possibly one of the tests in `tests/unistd/` is a natural place to add a test for this. Perhaps `misc.c`?